lib/presentation/settings/settings_screen.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../themes/constants.dart';
+
+class SettingsPage extends StatelessWidget {
+  SettingsPage({Key? key}) : super(key: key);
+  final List<Map<String, String?>> settingsContents = [
```

Comment:
TODO update URLs?
Handle contact and open source licenses in the app?

Reply:
Are the URLs available ??

Reply:
* Contact form screen (see demo contents)
* https://www.collaction.org/privacy
* https://www.collaction.org/terms
* Open-source libraries might not be that trivial. I was looking at [this plugin](https://pub.dev/packages/flutter_oss_licenses), but we also need to implement the design. Maybe we do not need to include this in the MVP.

Reply:
extracted to #110